ConanInstall.cmake

```diff
@@ -72,7 +72,7 @@ if(NOT CONAN_OPENSTUDIO_ALREADY_RUN)
     #boost_geometry/${BOOST_VERSION}@bincrafters/stable
     pugixml/1.10@bincrafters/stable
     jsoncpp/1.9.2
-    zlib/1.2.11
+    zlib/1.2.11@nrel/testing # TODO: Temp, pending merging of https://github.com/conan-io/conan-center-index/pull/1526, to resolve #3961
```

Comment:
Looks like this has been approved and just wanting on the merge.  https://github.com/conan-io/conan-center-index/pull/1526.  It'd be nice to have a ref to the official vs using the NREL so we can wait on this for another couple of days and check back.

Reply:
It won't be merged until I get approved in their Early access program, where access is granted by batches every two or three weeks from what I gathered after asking them directly. 

It also ties into #3940, at some point I'd just make this nrel/stable instead of nrel/testing and that's it.

Reply:
okay, well if that's the case we can make the change at later date.